app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -34,6 +42,14 @@ export class SearchAndAddOrder extends React.Component {
           <Tab
             tabName="InPatient">
             <SearchDrug />
+            {
+              this.props.drug.uuid ?
```

Comment:
Same as above.

Also, the two blocks of code are exactly the same. Consider extracting the code to a method.